### ISSUE TITLE
Docker README as steps

### ISSUE THREAD

**Issue by jpstroop** [loris-imageserver/loris#125]
@eliotjordan Would you be willing to organize the README for Docker into options and steps? As a newb, I'm not sure (I am now, mostly, but I'm pretending) if I'm supposed to do everything listed on that page, or choose certain headings based on what I think I want to do, or what.... 
So, maybe (as an outline):
- A sentence or two explaining what Docker can do for you
  - and then headings for each of those things, 
    1. and then steps (or whatever makes sense) for how to actually do it
  - (another thing it can do)
    1. (what to do)
    2. (what to do)
  - etc

Does that make sense?

**Comment by eliotjordan** [comment]
Yes it does. The README borrows the style of public docker images on [dockerhub](https://registry.hub.docker.com/u/dockerfile/nginx/) targeting a more experienced audience. I'll work on this.

**Comment by jpstroop** [comment]
Awesome, thanks.
